### PR TITLE
created text component

### PR DESCRIPTION
**Pull request by kbojarowska**
Possible sizes:
- small - 20px
- medium - 24px
- large - 26px
- x-large - 28px

### CODE REVIEW

File: skin-resque-frontend/src/App.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
+import React from 'react'
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/App.js:1]: Is the react import really needed?

File: skin-resque-frontend/src/components/Text/Text.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './Text.scss'
+
+export default function Text({size, children}) {
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/components/Text/Text.js:5]: Wouldn't typescript's types work here?

File: skin-resque-frontend/src/components/Text/Text.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
+@import url('https://fonts.googleapis.com/css2?family=Unna&display=swap');
+
+p {
+    font-family: 'Unna', serif;
+}
+
+.small {
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/components/Text/Text.scss:7]: Perhaps specify class to signify that it's meant for text? I think something like text-small or text-s would work

File: skin-resque-frontend/src/components/Text/Text.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    font-size: 20px;
+}
+
+.medium {
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/components/Text/Text.scss:11]: same as above

File: skin-resque-frontend/src/components/Text/Text.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    font-size: 24px;
+}
+
+.large {
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/components/Text/Text.scss:15]: same as above

File: skin-resque-frontend/src/components/Text/Text.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    font-size: 26px;
+}
+
+.x-large {
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/components/Text/Text.scss:19]: same as above

**Review by MrSpaceMan1 — CHANGES_REQUESTED** [review]
Minor suggestions. Resolve them and I'll send my approve

File: skin-resque-frontend/src/components/Text/Text.scss (hunk 3 above)
Comment (kbojarowska) [reply skin-resque-frontend/src/components/Text/Text.scss:7]: @dpionk what do you think about it?

Comment (dpionk) [reply skin-resque-frontend/src/components/Text/Text.scss:7]: I think it is good as it is

**Review by dpionk — APPROVED** [review]

**Review by MrSpaceMan1 — APPROVED** [review]
LGTM

**Pull request merged**